app/tests/payment/test_vipps_callback.py

```diff
@@ -0,0 +1,49 @@
+API_EVENT_BASE_URL = "/events/"
+
+
+def _get_registration_url(event):
+    return f"{API_EVENT_BASE_URL}{event.pk}/registrations/"
+
+
+def _get_registration_post_data(user, event):
+    return {
+        "user_id": user.user_id,
+        "event": event.pk,
+    }
+
+
+# these tests can not be tested because of celery
```

Comment:
Finne en løsning for dette

Reply:
Jeg har fått testet dette via frontend. Men skal få sett på det. Det kan hende at nå som jeg fikk fikset det med celery at testene også vil funke, siden tasken ikke kjører med en gang.

Reply:
Ser at man kan teste celery tasks med pytest og et bibliotek som heter pytest-celery. Hvordan opererer dere med å installere nye pakker i kodebasen?